setup.cfg

```diff
@@ -35,128 +35,6 @@ per-file-ignores =
-    ignore:(?s)Exception ignored in. <Finalize object, dead>.*sem_unlink.*FileNotFoundError:pytest.PytestUnraisableExceptionWarning
-    ignore:(?s)Exception ignored in. <coroutine object Scheduler\.add_worker.*in handle_worker.*Event loop is closed:pytest.PytestUnraisableExceptionWarning
-    ignore:(?s)Exception ignored in. <function Client\.__del__.*RuntimeError. IOLoop is closed:pytest.PytestUnraisableExceptionWarning
-    ignore:notifyAll\(\) is deprecated, use notify_all\(\) instead:DeprecationWarning:paramiko
-    ignore:setDaemon\(\) is deprecated, set the daemon attribute instead:DeprecationWarning:paramiko
-    ignore:`np.bool8` is a deprecated alias for `np.bool_`
-minversion = 6
-markers =
-    ci1: marks tests as belonging to 1 out of 2 partitions to run on CI ('-m "not ci1"' for second partition)
-    slow: marks tests as slow (deselected by default; select with '--runslow')
-    avoid_ci: marks tests as flaky or broken on CI on all OSs
-    ipython: marks tests as exercising IPython
-    gpu: marks tests we want to run on GPUs
-    leaking: ignore leaked resources; see pytest_resourceleaks.py for usage
-    workerstate: deterministic test for the worker state machine. Automatically applied to all tests that use the 'ws' fixture.
-
-# pytest-timeout settings
-# 'thread' kills off the whole test suite. 'signal' only kills the offending test.
-# However, 'signal' doesn't work on Windows (due to lack of SIGALRM).
-# The CI script modifies this config file on the fly on Linux and MacOS.
-timeout_method = thread
-# This should not be reduced; Windows CI has been observed to be occasionally
-# exceptionally slow.
-timeout = 300
-
-[mypy]
-# Silence errors about Python 3.9-style delayed type annotations on Python 3.8
-python_version = 3.9
-# See https://github.com/python/mypy/issues/12286 for automatic multi-platform support
-platform = linux
-# platform = win32
-# platform = darwin
-allow_incomplete_defs = false
-allow_untyped_decorators = false
-ignore_missing_imports = true
-no_implicit_optional = true
-show_error_codes = true
-warn_redundant_casts = true
-warn_unused_ignores = true
-warn_unreachable = true
-
-# FIXME must clean these modules up
-# Also look for 'mypy: disable-error-code' at the top of modules
-[mypy-distributed.client]
-allow_incomplete_defs = true
-[mypy-distributed.scheduler]
-allow_incomplete_defs = true
-[mypy-distributed.worker]
-allow_incomplete_defs = true
-
-# Recent or recently overhauled modules featuring stricter validation
-[mypy-distributed.active_memory_manager]
-allow_untyped_defs = false
-[mypy-distributed.system_monitor]
-allow_untyped_defs = false
-[mypy-distributed.worker_memory]
-allow_untyped_defs = false
-[mypy-distributed.worker_state_machine]
-allow_untyped_defs = false
-[mypy-distributed.shuffle.*]
-allow_untyped_defs = false
-[mypy-distributed.shuffle.tests.*]
-allow_untyped_defs = true
-
 [codespell]
 ignore-words-list = statics
 skip = docs/source/changelog.rst
```

Comment:
Think [`codespell` can be moved to `pyproject.toml`]( https://github.com/codespell-project/codespell/blob/a239c801c7bdece04f564ecbbc9b88e8261b4cca/README.rst#using-a-config-file ) as well